openapi.yaml

```diff
@@ -323,13 +323,15 @@ parameters:
     name: ownerId
     description: Unique Owner ID
     type: string
+    pattern: '[0-9]'
     required: true
   parkIdPath:
     name: parkId
     in: path
     description: Unique Park ID
     required: true
     type: string
+    pattern: '[0-9]'
```

Comment:
This pattern will match any string that contains at least one number. Could you change this so that it only matches strings that only contains numbers?